sources/world/GeoScribbles.geojson

```diff
@@ -0,0 +1,29 @@
+{
+    "geometry": null,
+    "properties": {
+        "attribution": {
+            "required": false,
+            "text": "GeoScribble contributors",
+            "url": "https://wiki.openstreetmap.org/wiki/GeoScribble"
+        },
+        "description": "Geometric survey notes, aged three weeks at most",
+        "i18n": false,
+        "id": "geoscribble-latest",
+        "available_projections": [
+            "EPSG:3857",
+            "EPSG:4326"
+        ],
+        "max_zoom": 22,
+        "name": "GeoScribble latest notes",
+        "type": "wms",
+        "default": false,
+        "overlay": true,
+        "transparent": true,
+        "category": "other",
+        "valid-georeference": true,
+        "url": "https://geoscribble.osmz.ru/?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=latest&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
```

Comment:
This is what this URL would look like in iD https://ideditor.netlify.app/#background=custom:https://geoscribble.osmz.ru/%3FFORMAT%3Dimage/png%26TRANSPARENT%3Dtrue%26VERSION%3D1.1.1%26SERVICE%3DWMS%26REQUEST%3DGetMap%26LAYERS%3Dlatest%26STYLES%3D%26SRS%3D{proj}%26WIDTH%3D{width}%26HEIGHT%3D{height}%26BBOX%3D{bbox}&disable_features=boundaries&map=15.95/52.4708/13.4178

I think this needs a background to work.

Reply:
Exactly: https://github.com/openstreetmap/iD/issues/9280

It does need background to work. Just like OSM GPS layer. That's why there's `"overlay": true` key included.

Reply:
Ah, I get it now, thanks. I never really looked at the overlay-prop and feature in iD but looking at the code now this should show up in the overlay section. Nice.